build.sbt

```diff
@@ -50,6 +50,43 @@ lazy val `memeid4s-http4s` = project
 lazy val `memeid4s-scalacheck` = project
   .dependsOn(memeid)
 
+lazy val bench = project
+  .dependsOn(memeid4s)
+  .enablePlugins(JmhPlugin)
```

Comment:
I think we should disable Scoverage plugin here so it doesn't affect global coverage